config/locales/gcse_details.yml

```diff
@@ -26,6 +26,11 @@ en:
         gcse: For example, ‘C’ or ‘4’
         gce_o_level: For example, ‘C’
         scottish_national_5: For example, ‘C’ or ‘4’
+  gcse_edit_year:
+    page_titles:
+      maths: When was your qualification awarded?
+      english: When was your qualification awarded?
+      science: When was your qualification awarded?
```

Comment:
Page titles for each subject seems redundant since they're all the same. Maybe we could reduce this to:

```yaml
gcse_edit_year:
  page_title: When was your qualification awarded?
```

Reply:
Good point, I forgot to change them to be specific!